# Goal
float32 target_x
float32 target_y
---
# Result
bool success
---
# Feedback
float32 progress